packages/pangea-node-sdk/src/services/audit.ts

```diff
@@ -573,6 +585,14 @@ class AuditService extends BaseService {
           publishedRoots: this.publishedRoots,
           record: record,
         });
+        if (record.consistency_verification === "fail") {
+          fixConsistencyProof(Number(record.leaf_index)).then(() => {
```

Comment:
Are these base10 integers? If so do this instead:
```suggestion
          fixConsistencyProof(parseInt(record.leaf_index, 10)).then(() => {
```